staging/src/kubevirt.io/client-go/apis/snapshot/v1alpha1/types.go

```diff
@@ -82,6 +82,10 @@ type VirtualMachineSnapshotStatus struct {
 
 	// +optional
 	Conditions []Condition `json:"conditions,omitempty"`
+
+	// +optional
+	// +listType=set
+	Indications []string `json:"indications,omitempty"`
```

Comment:
I wonder in indications should be an enumerated type?  It would be better for coding against the api but of course strings provide more flexibility.